src/less/components/payex/button.less

```diff
@@ -1,11 +1,14 @@
 @import "../button.less";
 
-&.btn {
-	font-family: var(--brand-default);
+.btn {
+	font-family: var(--brand-default-buttons);
 
+	&:active {
+		border: 3px solid @btn-active-border-color;
```

Comment:
see the other comment later in this file about trying not to introduce new variables, and to avoid LESS specific syntax in favor of pure CSS when possible